src/js/lib/content/src/index.html

```diff
@@ -22,12 +23,31 @@
         />
         <title>Enso</title>
         <link rel="stylesheet" href="/assets/style.css" />
+        <link type="text/css" rel="stylesheet" href="https://www.gstatic.com/firebasejs/ui/4.8.0/firebase-ui-auth.css" />
         <script type="module" src="/assets/index.js" defer></script>
         <script type="module" src="/assets/run.js" defer></script>
     </head>
     <body>
         <div class="titlebar"></div>
-        <div id="root"></div>
+        <div id="root">
+            <div id="auth-container">
+                <div class="auth-header">
+                    <h1>Sign in to Enso</h1>
+                    <div class="auth-text">
+                        <p>Enso lets you create interactive data workflows. In order to share them, you need an account. In alpha/beta versions, this account is required.</p>
+                    </div>
+                </div>
+                <div id="user-signed-out">
+                    <div id="firebaseui-container"></div>
+                </div>
+                <div id="user-email-not-verified" class="auth-info">
+                    Verification link is sent. You can sign in after verifying your email.
+                </div>
+            </div>
+            <div id="version-check" class="auth-info">
+                This version is no longer supported. Please download a new one.
+            </div>
+        </div>
```

Comment:
We are managing such things in JS. Including everything - loaders etc. Please create these divs from JS and remove them when not needed anymore.